helm/hello-world-app/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+- What is your name?
```

Comment:
Could also be a configmap. Might be more consistent for testing then? Uncertain.